.travis.yml

```diff
@@ -9,9 +9,10 @@ before_install:
     - sudo apt-get install python3-dev
```

Comment:
Was a [trial](https://github.com/zalando/ghe-backup/pull/43/commits/a2a125891c67229d747ce0162f405716934a888a) that was unfortunately not successful: https://travis-ci.org/zalando/ghe-backup/builds/201543299 the build is green, but the [ImportError](https://travis-ci.org/zalando/ghe-backup/builds/201543299#L382) should be fixed.
As of now, I don't have any idea why yaml package can't be imported.